src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/TestController.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.TestData
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.TestService
+
+@RestController
+@RequestMapping("/test", produces = [MediaType.APPLICATION_JSON_VALUE])
+class TestController(private val testService: TestService) {
+
```

Comment:
Defining a test endpoint, restricting access to particular roles that must exist on the provided token.